### PR TITLE
fixed minor bugs and updated images

### PR DESCRIPTION
**Pull request by volkb**

### CODE REVIEW

File: edit_user.php [hunk 1]
```diff
@@ -65,28 +65,30 @@
...
+                      echo "        <option value=\"volunteer\">Volunteer</option>";
+                      echo "        <option value=\"admin\">Admin</option>";
+                      echo "      </select>";
+                      echo "    </div>";
                   }
-
                 ?>
 
-                 <div class="text-center">
+                 <div class="form-group">
```
Comment (MichaelvMorrison) [inline edit_user.php]: Are you sure this shouldn't be <div class="form-group text-center">?

File: myforge.php [hunk 2]
```diff
@@ -127,19 +127,36 @@
           </div>
         </div>
       </div>";
+    }else if ($user_type == "lab"){
```
Comment (MichaelvMorrison) [inline myforge.php]: Did you log into each individual user type to make sure these all still look good?

Comment (volkb) [reply myforge.php]: yes

File: edit_user.php (hunk 1 above)
Comment (volkb) [reply edit_user.php]: that's what I thought at first too, but adding the text-center for whatever reason doesn't flush it to a new line, without it it does

File: myforge.php [hunk 3]
```diff
@@ -127,19 +127,36 @@
...
+        echo "<div class='row'>
+        <div class='col-xs-12 col-sm-8 col-md-6 mx-auto'>
+          <div class='card shadow-lg my-3'>
+            <div class='card-body'>
+            <h1 class='card-title text-center'>Welcome to the Forge</h1>
+            <p class='text-center'>What will you create today?</p>";
+        echo "<div class='row py-2'><div class='col-md-12 text-center'><a role=\"button\" class=\"btn btn-lg btn-warning btn-clock text-uppercase fixed-btn-size\" href='create_account.php'>Create Account</a></div></div>";
```
Comment (lramos15) [inline myforge.php]: Indent

**Pull request merged**